docbook/reference/en/en-US/modules/Async_Injection.xml

```diff
@@ -72,7 +72,7 @@ public class Foo {
 package my.test.asyc.resources;
 
 import io.reactivex.Single;
-import javax.ws.rs.ext.Provider;
+import jakarta.ws.rs.ext.Provider;
```

Comment:
We shouldn't change this yet.

Reply:
Hey @jamezp,

Actually, I realized that I'm probably working on the wrong branch. Doesn't it make sense to make changes like the above on a branch where we really are on Jakarta REST 3.0+? If the code is referring to javax.ws.rs.*, then what's the point of making any of these changes? 

Looking back at Alessio's agenda comment of September 8, he said, "Nov 16th is the hard deadline for removing any JavaEE compliance statement which is not explicitly referring to a past RESTEasy version." So, I think I've overreacted. Basically, everything I've done should go on a RESTEasy 6.0 branch, which we don't have. Can I take ee10 and merge main into it?

Anyway, it seems to me that for now, I should just look for any compliance statements.

-Ron